plugins/android/src/main/java/androidProjectConfiguration.kt

```diff
@@ -32,8 +30,9 @@ fun Project.androidProjectConfiguration(
         minSdk = minSdk,
         targetSdk = targetSdk,
         compileSdk = compileSdk,
-        kotlinVersion = kotlinVersion,
-        agpVersion = agpVersion,
+        // we don't need check properties for exist, we read it successfully in forma configuration
+        kotlinVersion = properties["forma.kotlinVersion"]!!.toString(),
```

Comment:
let's extract property names to constants

Reply:
Constants doest allowed, extracted in val